src/main/public/js/analytics-custom-event-sender.js

```diff
@@ -72,7 +72,7 @@ $(function () {
   })
 
   // Send a google analytics event when a form that has the 'analytics-click-event-trigger' class is submitted.
-  // Example <form method="post" class="analytics-submit-event-trigger" data-event-action="Your GA action" data-event-label-from="Form element to extract GA label from"></form>
+  // Example <form method="post" class="analytics-submit-event-trigger" data-event-action="Your GA action" data-event-label-from="Form element to extract GA label minRange"></form>
```

Comment:
Please revert, that change has nothing to do with new fees API.

Reply:
How did this change 😄 ? Must be intelliJ's fault. Well spotted 👍